Lib/fontTools/varLib/__init__.py

```diff
@@ -620,6 +620,21 @@ def normalize(name, value):
 )
 
 
+def _add_CFF2(varFont, model, master_fonts):
+	from fontTools.cffLib.cff2_merge_funcs import (convertCFFtoCFF2, 
+								addCFFVarStore, 
+								merge_region_fonts)
+	glyphOrder = varFont.getGlyphOrder()
+	convertCFFtoCFF2(varFont)
+	# I sort the master font in the model order so as to
+	# not have to index through model.mapping when building
+	# a blend vector from the master font data.
+	ordered_fonts_list = [master_fonts[idx] for idx in model.reverseMapping]
+	model.mapping = model.reverseMapping = range(len(master_fonts))
```

Comment:
Maybe we should add a method `VariationModel.reorderMasters()`, so we don't tweak its internal data?

Reply:
Sounds good to me. Will do.

Reply:
It should, in the VariationModel, also reorder self.origLocations, and possibly clear self._subModels or something.